microarray/src/org/labkey/microarray/MicroarrayRunType.java

```diff
@@ -20,12 +20,12 @@
 import org.labkey.api.data.ButtonBar;
 import org.labkey.api.data.ContainerFilter;
 import org.labkey.api.exp.ExperimentRunType;
-import org.labkey.api.exp.Lsid;
+//import org.labkey.api.exp.Lsid;

```

Comment:
Just delete these imports if they're not needed, no reason to keep them around as comments.

Reply:
Oops, I intended to. Thanks for this catch.